### DISCUSSION TITLE
[N.E.P.S.][有二次设定]新艾利都治安局对外公开的警员资料（部分）

### DISCUSSION THREAD

**Discussion by RWHwaxili** [share121/inter-knot#5296] [其他]
### 朱鸢
新艾利都治安局光映分局 空洞事务科科长
警号001014 二级警督 副处级
艾利都大学毕业 学生会副主席 优秀毕业校友
### 青衣
新艾利都治安局光映分局 空洞事务科科员
兼新艾利都雅努斯区第一中学 法治副校长
警号001023 一级警司 副科级
（没学历因为是机器人）
### 赛斯
新艾利都治安局光映分局 空洞事务科科员
警号001057 一级警员 办事员
新艾利都航空大学法律系硕士

**Comment by chenyunji** [comment]
目前可公开情报（）

**Comment by saitou-yuki** [comment]
- 前営業日にやったこと
  - 定常タスク
    - PRレビュー
  - https://github.com/daytona-international/di-docs/issues/2838
    - アプリBFF作成に向けてプロンプト調整
      - 商品詳細など観点膨大なものは手動でexcel作成予定
      - 途中
  - https://github.com/daytona-international/di-ddp/pull/28921
    - pre-staging3で成功したらproductionに入れる
    - DBスペック不足、保留中：https://github.com/daytona-international/di-ddp/pull/28909#issuecomment-3845146457
    - 途中
  - ネイティブアプリ
    - https://github.com/daytona-international/di-ddp-member/pull/2804
    - https://github.com/daytona-international/di-ddp-member/pull/2806
- 今日やること
  - 定常タスク
    - [ ] PRレビュー
  - [ ] https://github.com/daytona-international/di-docs/issues/2838
    - [ ] アプリBFF作成に向けてプロンプト調整
      - 商品詳細など観点膨大なものは手動でexcel作成予定
  - [ ] https://github.com/daytona-international/di-ddp/pull/28921
    - pre-staging3で成功したらproductionに入れる
    - DBスペック不足、保留中：https://github.com/daytona-international/di-ddp/pull/28909#issuecomment-3845146457
  - [ ] https://github.com/daytona-international/di-ddp/pull/28654
    - https://github.com/daytona-international/di-ddp/pull/28921 リリース後にstaging検証
  - ネイティブアプリ
    - [ ] https://github.com/daytona-international/di-docs/issues/2884
    - [ ] https://github.com/daytona-international/di-docs/issues/3021
    - [ ] https://github.com/daytona-international/di-docs/issues/2885
    - 時間あればやる
      - [ ] https://github.com/daytona-international/di-bff/issues/2791
      - [ ] https://github.com/daytona-international/di-bff/issues/2797
  - 保留中
    - [ ] ZETA_SEARCH:価格帯検索時の参照項目について確認
      - [To: ZETA様](https://daytonajp.backlog.com/view/APP_RENEWAL-2842)
- 困っていること
- 滞留PR報告
  - di-ddp
    - https://github.com/daytona-international/di-ddp/pull/28911
  - di-ddp-member
    - https://github.com/daytona-international/di-ddp-member/pull/2804
    - https://github.com/daytona-international/di-ddp-member/pull/2806
  - di-docs
  - di-bff
  - di-ddp-sales
    - リリース依頼
      - https://github.com/daytona-international/di-ddp-sales/milestone/135
